.circleci/test-integration.sh

```diff
@@ -6,18 +6,28 @@ mkdir -p test-results/js
 echo "`date` Running integration tests in $WHYIS_DEMO_IMAGE"
 #JS_REDIRECT=""
 #docker run $WHYIS_DEMO_IMAGE bash -c "mkdir -p /apps/whyis/test-results/js && curl -sL https://deb.nodesource.com/setup_12.x | bash - $JS_REDIRECT && apt-get install -y nodejs xvfb libgtk-3-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 $JS_REDIRECT && cd /apps/whyis/tests/integration && npm install $JS_REDIRECT && CYPRESS_baseUrl=http://localhost npm run cypress:run $JS_REDIRECT"
-JS_REDIRECT=">/apps/whyis/test-results/js/test.out 2>/apps/whyis/test-results/js/test.err"
-docker run $WHYIS_DEMO_IMAGE bash -c "mkdir -p /apps/whyis/test-results/js && curl -sL https://deb.nodesource.com/setup_12.x | bash - $JS_REDIRECT && apt-get install -y nodejs xvfb libgtk-3-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 $JS_REDIRECT && cd /apps/whyis/tests/integration && npm install $JS_REDIRECT && CYPRESS_baseUrl=http://localhost npm run cypress:run-ci $JS_REDIRECT; cp -p -R cypress/screenshots /apps/whyis/test-results/js; cp -p -R cypress/videos /apps/whyis/test-results/js; cd /apps/whyis && tar cf test-results-js.tar test-results/js && cat test-results-js.tar" >test-results-js.tar
+#JS_REDIRECT=">/apps/whyis/test-results/js/test.out 2>/apps/whyis/test-results/js/test.err"
+#docker run $WHYIS_DEMO_IMAGE bash -c "mkdir -p /apps/whyis/test-results/js && curl -sL https://deb.nodesource.com/setup_12.x | bash - $JS_REDIRECT && apt-get install -y nodejs xvfb libgtk-3-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 $JS_REDIRECT && cd /apps/whyis/tests/integration && npm install $JS_REDIRECT && CYPRESS_baseUrl=http://localhost npm run cypress:run-ci $JS_REDIRECT; cp -p -R cypress/screenshots /apps/whyis/test-results/js; cp -p -R cypress/videos /apps/whyis/test-results/js; cd /apps/whyis && tar cf test-results-js.tar test-results/js && cat test-results-js.tar" >test-results-js.tar
+
+#ERR_REDIRECT="2>/apps/whyis/test-results/js/test.err"
+
+# docker network create whyis-network
+# docker run --detach --name whyis $WHYIS_DEMO_IMAGE
+# docker run --network container:whyis --name whyis-integration tetherlessworld/whyis-integration:latest
+
+# docker run --name whyis-demo $WHYIS_DEMO_IMAGE bash -c "mkdir -p /apps/whyis/test-results/js && curl -sL https://deb.nodesource.com/setup_12.x | bash - && apt-get install -y nodejs && cd /apps/whyis/tests/integration && npm install && CYPRESS_baseUrl=http://localhost npm run cypress:run-ci && tar cf test-results-js.tar results"
+
+docker-compose -f docker/compose/testing/docker-compose.yml up --abort-on-container-exit 
+docker cp whyis-integration:/integration/test-results-js.tar test-results/js/
```

Comment:
Results probably aren't available yet, here. Need to loop until the file exists or some time has elapsed.
See e.g., the curl + sleep loops I've done before.

Reply:
I was under the impression that docker-compose should block until the tests are over, which by then, results should exist. `tar` doesn't seem to complain on CircleCI, so I thought the result tar should be there.

Reply:
No, I don't think that's the case. Tar generally doesn't complain if you pass it nothing.